22-discovery.ks

```diff
@@ -31,45 +37,83 @@ echo " * enabling required system services"
 systemctl enable ipmi.service
 systemctl enable foreman-proxy.service
 systemctl enable discovery-fetch-extensions.path
+systemctl enable discovery-menu.service
+
+# register service is started manually from discovery-menu
+systemctl disable discovery-register.service
+
+echo " * disabling some unused system services"
+systemctl disable ipmi.service
+
+echo " * open foreman-proxy port via firewalld"
+firewall-offline-cmd --zone=public --add-port=8443/tcp --add-port=8448/tcp
```

Comment:
Wait, ssh port is enabled by default, no need to add it explicitly (Anaconda does that by default).